sklearn/semi_supervised/_label_propagation.py

```diff
@@ -105,6 +105,34 @@ class BaseLabelPropagation(ClassifierMixin, BaseEstimator, metaclass=ABCMeta):
          ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
          ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
          for more details.
+
+    Attributes
```

Comment:
I don't know what the convention is -> unsure if the extra docs on `BaseLabelPropagation` make sense as users can't directly import/access this class.

Does someone know?

Reply:
Traditionally, we only include Attribute docs for public classes. For example, `BaseDecisionTree` does not include any docs for Attributes because it is private:

https://github.com/scikit-learn/scikit-learn/blob/75db1bc2dbd716016170241d8daae97624d01252/sklearn/tree/_classes.py#L90

In this case, since `BaseLabelPropagation` is a private class, I would not include the Attributes section here.